pages/_app.tsx

```diff
@@ -1,7 +1,15 @@
 import '../styles/globals.css'
+import NavBar from '../src/components/NavBar/NavBar';
 
 function MyApp({ Component, pageProps }) {
-  return <Component {...pageProps} />
+<<<<<<< HEAD
```

Comment:
here's a conflict with the master branch and the branch you're trying to merge in. accept current change keeps the changes from master and accept incoming change will keep the changes from your branch